oneapp-server/types/ApplicationProgramInfo.js

```diff
@@ -0,0 +1,52 @@
+const { gql } = require('apollo-server-express');
+
+const typeDef = gql`
+  extend type Application {
+    programInfo: ApplicationProgramInfo
+  }
+
+  type ApplicationProgramInfo {
+    "Applying for SNAP"
+    IS_FS_SELECTED: Boolean
+    "Applying for TANF"
+    IS_TF_SELECTED: Boolean
+    "Applying for General Assistance"
+    IS_GA_SELECTED: Boolean
+    "Has active SNAP case"
+    HAVE_ACTIVE_CASE_CURRENTLY: Boolean
+    "Case number(s) of active SNAP applications"
+    CURRENT_CASE_NUMBERS: String
+    "Has received SNAP previously"
+    HAD_ACTIVE_CASE_PREVIOULSY: Boolean
```

Comment:
This typo is in the database. Opting to replicate the typo in the GraphQL type so db fields map to GraphQL types 1:1.